src/app/api/mock/onerep/profiles/[profileId]/activate/route.ts

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest, NextResponse } from "next/server";
+import { errorIfProduction } from "../../../../utils/errorThrower";
+
+export function PUT(req: NextRequest) {
+  const prodError = errorIfProduction();
+  if (prodError) return prodError;
+
+  const profileId: number = Number(req.url.match(/profiles\/([0-9]+)/)![1]);
```

Comment:
Since you're using `[profileId]` in the file path, this is already parsed for you: https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes

e.g. if you do:
```
export async function PUT(
  req: NextRequest,
  { params }: { params: { profileId: number } },
) 
```

Then `params.profileId` will already be a parsed `Number` and you don't need to do the extra work here.